test/simpleTestForGenerator/CC-BY-NC-ND-3.0-IGO.txt~

```diff
@@ -0,0 +1,98 @@
+Attribution-NonCommercial-NoDerivs 3.0 IGO
```

Comment:
It looks like the `.txt~` file got included here inadvertently -- can you remove that from the PR?

Reply:
Thanks, Steve! All fixed in the latest commit.